packages/dbml-core/src/parse/dbml/parser.pegjs

```diff
@@ -183,6 +183,21 @@ ref_body
       settings: ref_settings
     };
   }
+//CHANGE
+RefField
+  = field:(RefSingleField/RefMultipleFields) { 
+    if (typeof field === "string") field = [field];
+    return field; 
+  }
+
+RefSingleField
+  =  field:name { return field; }
+ 
+RefMultipleFields
+  = "(" sp* first:RefSingleField rest:(Comma sp* RefSingleField)* ")"  {
```

Comment:
This input will fail.
`users.(id , name  ) < orders.(  user_id , username  )`

Suggest adding `sp*` before Comma and after `rest` here.

Reply:
Also, pls change `fieldName` to `fieldNames`